### PR TITLE
Cosmetic url replacement

### PR DESCRIPTION
**Pull request by CGarces**
Change url inside text to https://libreelec.tv

### CODE REVIEW

File: distributions/LibreELEC/options [hunk 1]
```diff
@@ -13,7 +13,7 @@
 # Welcome Message for e.g. SSH Server (up to 5 Lines)
   GREETING0="##############################################"
   GREETING1="#                 LibreELEC                  #"
-  GREETING2="#            http://libreelec.tv             #"
+  GREETING2="#            https://libreelec.tv             #"
```
Comment (MilhouseVH) [inline distributions/LibreELEC/options]: Can you remove one of the trailing spaces to keep the hashes aligned?

Comment (CGarces) [inline distributions/LibreELEC/options]: done

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -104,8 +104,8 @@ Please follow this process; it's the best way to get your work included in the p
 - [Open a Pull Request](https://help.github.com/articles/using-pull-requests) with a
     clear title and description.
 
-[Forum]: http://libreelec.tv/forum
+[Forum]: https://libreelec.tv/forum
```
Comment (CvH) [inline CONTRIBUTING.md]: https://forum.libreelec.tv/

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -104,8 +104,8 @@ Please follow this process; it's the best way to get your work included in the p
...
 [issue]: https://github.com/LibreELEC/LibreELEC.tv/issues
-[webchat]: http://libreelec.tv/support/chat
+[webchat]: https://libreelec.tv/support/chat
```
Comment (CvH) [inline CONTRIBUTING.md]: the url goes into the nowhere, @chewitt fix or remove ?

Comment (chewitt) [inline CONTRIBUTING.md]: remove it please

Comment (CGarces) [inline CONTRIBUTING.md]: Done

**Pull request merged**